cockatrice/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ SET(cockatrice_SOURCES
     src/dlg_edit_password.cpp
     src/dlg_edit_tokens.cpp
     src/dlg_edit_user.cpp
+	src/dlg_forgotpassword.cpp
```

Comment:
Currently all the code is indented using 4 spaces, this PR is using tabs instead.